packages/web-twig/src/Resources/components/CheckboxField/stories/CheckboxFieldValidation.twig

```diff
@@ -0,0 +1,33 @@
+<CheckboxField
+    id="checkboxfieldSuccess"
+    label="Checkbox Label"
+    name="checkboxfieldSuccess"
+    validationState="success"
+/>
+
+<CheckboxField
+    id="checkboxfieldWarning"
+    label="Checkbox Label"
+    validationText="Warning validation text"
+    name="checkboxfieldWarning"
+    validationState="warning"
+/>
+
+<CheckboxField
+    id="checkboxfieldDanger"
+    isChecked
```

Comment:
We don't have this one checked in the `web` package…

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/56a15742-e303-4c55-a3f2-7136dadb1ac6)

Reply:
Fixed.